demo/chbench/mzcompose.yml

```diff
@@ -192,11 +192,11 @@ services:
   mzutil:
     mzbuild: mzutil
     volumes:
-      - ${MZ_CHBENCH_SNAPSHOT:-mzutil}:/snapshot
+      - ${MZ_CHBENCH_SNAPSHOT:-kafka-data}:/snapshot
```

Comment:
Last suggested change, I promise. Perhaps instead of calling this `kafka-data`, we can call this `mzsnap-data` so that we don't accidentally conflict on file / directory names with Kafka?

Reply:
yeah fair, done.